### PR TITLE
Add TinkerPop

### PR DESCRIPTION
**Pull request by mohataher**
Added [awesome-tinkerpop](https://github.com/mohataher/awesome-tinkerpop) to databases section.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Some feedback:

- Instead of "The main idea is that everyone can contribute here, so we can center all the informations about TinkerPop and keep up-to-date.", use that space to succinctly explain what TinkerPop is.
- There should be a `-` between the link and description for each item.
- Be consistent about capitalization of headings.
- `Awesome-TinkerPop` => `Awesome TinkerPop`
- `StackOverflow` => `Stack Overflow`
- Be consistent about `TinkerPop3` or `TinkerPop 3`. Pick one.
- I'd like to see more tutorials, resources, talks, videos, books, etc.
- Use sub-headings `###` for the `TinkerPop 3 libraries` sub-items, not sub-lists. 
- You have a link that is just named link `link`.
- Be consistent about ending all descriptions in a `.`

File: readme.md [hunk 1]
```diff
@@ -310,6 +310,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Doctrine](https://github.com/TomasVotruba/awesome-doctrine) - PHP ORM
 - [MongoDB](https://github.com/ramnes/awesome-mongodb)
 - [RethinkDB](https://github.com/d3viant0ne/awesome-rethinkdb)
+- [TinkerPop](https://github.com/mohataher/awesome-tinkerpop)
```
Comment (sindresorhus) [inline readme.md]: Add a very short description here on what TinkerPop is.

**Comment by mohataher** [comment]
Thanks for the feedback. I have done each one of them except adding resources. There are no more `awesome` resources available. I will add more as I see them.

File: readme.md [hunk 2]
```diff
@@ -310,6 +310,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Doctrine](https://github.com/TomasVotruba/awesome-doctrine) - PHP ORM
 - [MongoDB](https://github.com/ramnes/awesome-mongodb)
 - [RethinkDB](https://github.com/d3viant0ne/awesome-rethinkdb)
+- [TinkerPop](https://github.com/mohataher/awesome-tinkerpop) Apache TinkerPop™ is a graph computing framework.
```
Comment (sindresorhus) [inline readme.md]: Follow the existing style for link description.

It's enough with:

> Graph computing framework

**Comment by mohataher** [comment]
Just updated the PR.

**Comment by sindresorhus** [comment]
> TinkerPop2 to TinkerPop3. #### Wrappers/Clients

🐛⬆️

> StackOverflow => Stack Overflow

Not done.

> Be consistent about TinkerPop3 or TinkerPop 3. Pick one.

Repo description too.

**Comment by mohataher** [comment]
All changes made. Sorry about that.

I left TinkerPop 2 as it's since it's a standard to write it like `TinkerPop 2` with a space in between.

**Pull request closed without merging**